sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/sampleoverview/SampleView.groovy

```diff
@@ -0,0 +1,78 @@
+package life.qbic.portal.sampletracking.components.sampleoverview
+
+import com.vaadin.data.provider.ListDataProvider
+import com.vaadin.shared.ui.grid.HeightMode
+import com.vaadin.ui.Grid
+import life.qbic.business.samples.Sample
+import life.qbic.business.samples.info.GetSamplesInfoOutput
+import life.qbic.portal.sampletracking.communication.notification.NotificationService
+import life.qbic.portal.sampletracking.components.ViewModel
+
+class SampleView extends SampleDesign{
+
+    private final ViewModel viewModel
+    private final Presenter presenter
+
+
+    SampleView(ViewModel viewModel, NotificationService notificationService) {
+        super()
+        this.viewModel = viewModel
+        this.presenter = new Presenter(notificationService, viewModel)
+
+
+        addClickListener()
+        createSamplesGrid()
+    }
+
+    private void addClickListener() {
```

Comment:
As this method name is used in the framework differently I would refrain from calling this method like this.
```suggestion
    private void activateViewToggle() {
```